_posts/en/2016-xx-xx-Vuejs-makes-a-copy-update-replace-inside-the-data-binding.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+
+title: Vuejs makes a copy-upate-replace inside the data binding
```

Comment:
Update to "VueJS: How VueJS makes a copy-update-replace inside the data binding."